### DISCUSSION TITLE
Kohya with Python 3.10.11

### DISCUSSION THREAD

**Discussion by Sparhawker** [bmaltais/kohya_ss#2170] [Q&A]
Hi people,

I tried to install kohya. But it needs python 3.10.11 version. I am using Forge and reactor sometimes they are working with 3.10.6. when I upgrade pyton version lots extensions broken (civitai helper, reactor vs...) Is there another way to walk around. or am I missing something?

**Comment by Jennal** [comment]
You can create virtual enviroment for both projects. Virtual enviroment can base on folder, then you can use different python version in different project.

Try pyenv(https://github.com/pyenv/pyenv), or any other virtual enviroment provider you can find.

**Comment by Sparhawker** [comment]
Thank you for your answer. I will try
